libs/libzauth/libzauth/Cargo.toml

```diff
@@ -13,6 +13,15 @@ rustc-serialize = ">= 0.3"
 sodiumoxide     = "^0.2.7"
 regex           = "1.6"
 lazy_static     = "1.4"
+jwt-simple      = "0.11"
+serde_json      = "1.0"
+serde           = "1.0"
+thiserror       = "1.0"
+base64          = "0.21"
 
 [dev-dependencies]
 clap = ">= 2.0"
+
+[patch.crates-io.jwt-simple]
```

Comment:
```
[dependencies.jwt-simple]
git = "https://github.com/wireapp/rust-jwt-simple"
rev = "commit_id"
```

Also, can you add a comment stating why this was forked (what's missing from upstream), and did we send it along already?